packages/pirania/files/etc/config/pirania

```diff
@@ -18,5 +18,8 @@ config base_config 'base_config'
     list allowlist_ipv6 'fc00::/7'
     list allowlist_ipv6 'fe80::/64'
     list allowlist_ipv6 '2a00:1508:0a00::/40'
-    list catch_interfaces 'br-lan'
-    list catch_interfaces 'anygw'    
+    option catch_virtual_ifaces 'yes'
+    list virtual_ifaces_catched 'br-lan'
+    list virtual_ifaces_catched 'anygw'
+    option catch_wifi_mode_ifaces 'no'
+    list wifi_mode_ifaces_catched 'wlan0-ap'
```

Comment:
AFAIU, the problem is not "wifi mode" but simply that the interface is part of a bridge. then you need ebtables to filter the packets.

so the name of this setting is a bit misleading to me,

maybe `catch_bridged_interfaces`?